src/Swashbuckle.AspNetCore.SwaggerGen/Generator/SwaggerGenerator.cs

```diff
@@ -25,20 +25,23 @@ public class SwaggerGenerator : ISwaggerProvider
         }
 
         public SwaggerDocument GetSwagger(
-            string documentName,
+            string documentVersion,
```

Comment:
I have thought about using this for multi-tenancy where you want to do custom work that leads to a contract change